templates/webapps/galaxy/workflow/display.mako

```diff
@@ -30,6 +30,15 @@
       <% new_prefix = prefix + input.name + "|" %>
       ${row_for_param( input.test_param, group_values[ input.test_param.name ], other_values, prefix, step )}
       ${do_inputs( input.cases[ current_case ].inputs, group_values, new_prefix, step, other_values )}
+    %elif input.type == "section":
+      <% new_prefix = prefix + input.name + "|" %>
+      <% group_values = values[input.name] %>
+      <div class="form-title-row"><b>${input.title}:</b></div>
+      <div class="repeat-group">
+        <div class="repeat-group-item">
```

Comment:
@nsoranzo required for CSS to work properly. If someone has made changes to styling on repeats, I would like sections to appear the same.